pkgs/test_core/lib/src/runner/engine.dart

```diff
@@ -66,6 +69,9 @@ class Engine {
   /// `false` if the tests finished running before close was called.
   bool _closedBeforeDone;
 
+  /// The coverage output directory
```

Comment:
[nit] Format as a sentence

```suggestion
  /// The coverage output directory.
```

Reply:
Done!